frontends/web/src/locales/en/app.json

```diff
@@ -1544,7 +1544,8 @@
     "dashboard": {
       "allSessions": "All sessions",
       "newConnection": "New connection",
-      "noConnectedSessions": "No accounts are currently connected to any dapps."
+      "noConnectedSessions": "No accounts are currently connected to any dapps.",
+      "disclaimer": "Walletconnect is a protocol to connect to Ethereum based Dapps. These dapps are run by third-party services, so only connect to dapps you trust and be sure to always know what you are signing when making a transaction."
```

Comment:
CI fails, this should be sorted alphabetically. you can also run `npx locize-cli format frontends/web/src/locales --format json` to fix it.